### PR TITLE
Enhance - Ticket Emails display Event excerpt within Email Preview.

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[TE-076D]

Screenshot: https://share.cleanshot.com/2ndjp5p4

### CODE REVIEW

**Review by redscar — APPROVED** [review]

File: src/views/integrations/event-tickets/emails/template-parts/body/event/description.php [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 ?>
 <tr>
-	<td style="padding:0;" class="tec-tickets__email-table-content-event-description-container">
+	<td style="padding-bottom:20px;" class="tec-tickets__email-table-content-event-description-container">
```
Comment (juanfra) [inline src/views/integrations/event-tickets/emails/template-parts/body/event/description.php:30]: I think for consistency, if we're moving the post title to ET, we should also do it for the excerpt.

Comment (codingmusician) [reply src/views/integrations/event-tickets/emails/template-parts/body/event/description.php:30]: The styling should be moved into the `styles.php` file located in the `template-parts/header/head/` folder.

**Review by codingmusician — APPROVED** [review]
Approved with the understanding that this will be a temporary solution.

**Comment by rafsuntaskin** [comment]
Solved with: https://github.com/the-events-calendar/the-events-calendar/pull/4287

**Pull request closed without merging**